wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/ProfileDeviceDetailsView.swift

```diff
@@ -0,0 +1,223 @@
+                        },
+                        label: {
+                            Image(.backArrow)
+                                .renderingMode(.template)
+                                .foregroundColor(SemanticColors.Icon.foregroundDefaultBlack.swiftUIColor)
+                        }
+                    )
+                }
+                ToolbarItem(placement: .principal) {
+                    DeviceView(viewModel: viewModel).titleView
+                }
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    SwiftUI.Button(
+                        action: {
+                            isDebugViewPresented.toggle()
+                        },
+                        label: {
+                            if viewModel.showDebugMenu {
+                                Text("Debug")
+                            }
+                        }
+                    )
+                }
+            }
+        }
+
+        .background(SemanticColors.View.backgroundDefault.swiftUIColor)
+        .navigationBarBackButtonHidden(true)
+        .onAppear {
+            viewModel.onAppear()
+        }
+        .onDisappear {
+            dismissedView?()
+        }
+        .onReceive(viewModel.$shouldDismiss) { shouldDismiss in
+            if shouldDismiss {
+                dismiss()
+            }
+        }
+        .sheet(isPresented: $isCertificateViewPresented) {
+            if let certificate = viewModel.e2eIdentityCertificate {
+                E2EIdentityCertificateDetailsView(
+                    certificateDetails: certificate.details,
+                    isDownloadAndCopyEnabled: viewModel.isCopyEnabled,
+                    isMenuPresented: false,
+                    performDownload: viewModel.downloadE2EIdentityCertificate,
+                    performCopy: viewModel.copyToClipboard
+                )
+            }
+        }
+        .alert("Debug options", isPresented: $isDebugViewPresented, actions: {
+            SwiftUI.Button("Delete Device", action: {
+                  viewModel.onDeleteDeviceTapped()
+              })
+            SwiftUI.Button("Duplicate Session", action: {
+                  viewModel.onDuplicateClientTapped()
+              })
+            SwiftUI.Button("Corrupt Session", action: {
+                viewModel.onCorruptSessionTapped()
+            })
+            SwiftUI.Button("Cancel", role: .cancel, action: {
+                isDebugViewPresented.toggle()
+            })
+            }, message: {
+              Text("Tap to perform an action")
+            })
+    }
+
+    @ViewBuilder
+    func sectionTitleView(title: String, description: String? = nil) -> some View {
+        Text(title)
+            .font(FontSpec.mediumRegularFont.swiftUIFont)
+            .foregroundColor(SemanticColors.Label.textSectionHeader.swiftUIColor)
+            .padding([.leading, .top], ViewConstants.Padding.standard)
+        if let description = description {
```

Comment:
💅 having some blank lines in SwiftUI usually helps to make it easier to read, at least for me :)

Reply:
done